how-to/workspace-platform-starter/client/src/framework/snap.ts

```diff
@@ -0,0 +1,328 @@
+import type OpenFin from "@openfin/core";
+import * as Snap from "@openfin/snap-sdk";
+import { getApp } from "./apps";
+import { launchAppAsset, launchExternal } from "./launch";
+import { createLogger } from "./logger-provider";
+import { MANIFEST_TYPES } from "./manifest-types";
+import * as platformSplashProvider from "./platform/platform-splash";
+import type { SnapProviderOptions } from "./shapes";
+import { formatError, isEmpty } from "./utils";
+
+const logger = createLogger("Snap");
+const NATIVE_APP_PREFIX = "@app";
```

Comment:
Only thing that concerns me about using @ as part of the app identity is that @ is used in fdc3 app directory naming e.g. app1@host.appd.com. I know this starts with @app but maybe something else that avoids confusion e.g. *app

Reply:
Done